.storybook/main.js

```diff
@@ -1,11 +1,7 @@
 /** @type {import('@storybook/react/types').StorybookConfig} */
 module.exports = {
   stories: ["../easy-ui-react/src/**/*.stories.mdx"],
-  addons: [
-    "@storybook/addon-links",
-    "@storybook/addon-essentials",
-    "@storybook/addon-interactions",
-  ],
+  addons: ["@storybook/addon-essentials", "@storybook/addon-a11y"],
```

Comment:
if in the future if we find we want the other addons we can add them them. didn't want to pollute our addons with unused ones